lib/ansible/config/base.yml

```
@@ -946,9 +946,10 @@ DEFAULT_MANAGED_STR:
   - {key: ansible_managed, section: defaults}
   yaml: {key: defaults.ansible_managed}
   deprecated:
-    why: The `ansible_managed` variable can be set just like any other variable, or a different variable can be used.
-    version: "2.23"
-    alternatives: Set the `ansible_managed` variable, or use any custom variable in templates.
+    why: The 'ansible_managed' setting in ansible.cfg is deprecated. Its name was confusing and it's better to set 'ansible_managed' as a normal variable.
```

Plz keep the backticks. They are meaningful in RST. They should probably be doubled though.
```suggestion
    why: The ``ansible_managed`` setting in ``ansible.cfg`` is deprecated. Its name was confusing and it's better to set ``ansible_managed`` as a normal variable.
```